tests/run/assets/simple.py

```diff
@@ -12,5 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+import mlrun
+
+
 def handler(context, x=1):
     context.log_result("y", x * 2)
+
+
+def func():
```

Comment:
can make the handler name a bit more meaningful
```suggestion
def get_ctx_kind_label():
```